lib/plaid_http.ex

```diff
@@ -0,0 +1,54 @@
+defmodule PlaidHTTP do
```

Comment:
```suggestion
defmodule PlaidHTTP do
```

Otherwise you are taking an extra namespace from `Plaid` itself

Reply:
I updated this in the latest commit.